libs/developers-api-client/src/main/kotlin/com/linecorp/link/developers/client/response/responses.kt

```diff
@@ -345,3 +350,17 @@ data class ItemTokensMediaResourceStatus(
 data class UpdateTokenMediaRefreshResponse(
     val requestId: String
 )
+
+
+data class TxMessageListResponse(
+    val messages: List<TxMessageWithDetailResponse>,
+    val nextPageToken: String,
+    val prevPageToken: String
+)
+
+data class TxMessageWithDetailResponse(
+    val msgIndex: Int,
+    val requestType: String,
+    val details: Any,
+)
```

Comment:
We need unify style:

style 1: always have `,` in the last param:
```kotlin
data class TxMessageListResponse(
    val messages: List<TxMessageWithDetailResponse>,
    val nextPageToken: String,
    val prevPageToken: String,
)

data class TxMessageWithDetailResponse(
    val msgIndex: Int,
    val requestType: String,
    val details: Any,
)
```

style 2: nerver have `,` in the last param:

```
data class TxMessageListResponse(
    val messages: List<TxMessageWithDetailResponse>,
    val nextPageToken: String,
    val prevPageToken: String
)

data class TxMessageWithDetailResponse(
    val msgIndex: Int,
    val requestType: String,
    val details: Any
)
```

Reply:
Fixed at b886a12351f3bf5a274a48cbddd4820cf53a7869